docs/advanced.md

```diff
@@ -9,12 +9,17 @@ all outgoing requests.
 !!! hint
     A Client instance is equivalent to a Session instance in `requests`.
 
+!!! note
+    Starting from `httpx==0.10.0`, the default and recommended Client class is `AsyncClient`.
```

Comment:
Should we note that it’s because we’re hoping to reintroduce sync+async support shortly, or something along those lines?

Reply:
Sure, added a note about that. :-)